back/engines/commercial/multi_tenancy/lib/tasks/core/clean_up_orphaned_image_records.rake

```diff
@@ -25,20 +31,48 @@ namespace :cl2back do
         # is eventually referenced by a layout (when / if the layout is saved).
         # By only destroying unused layout_images with an age of 3+ days, we can be reasonably confident that the
         # admin does not intend to add the image to a layout, and the image is truly orphaned.
-        if image.created_at < 3.days.ago
-          image.destroy! if live_run
-          puts " destroyed layout_image with unused code field: #{image.id}"
-        end
+        next if image.created_at > 3.days.ago
+
+        image.destroy! if live_run
+        n_li_destroyed += 1
+        puts " destroyed layout_image with unused code field: #{image.id}"
       end
 
       # text_images
       # Destroy text_image if ref not found anywhere in associated imageable record.
       TextImage.all.includes(:imageable).each do |image|
-        unless image.imageable.to_json.include?(image.text_reference)
-          image.destroy! if live_run
-          puts " destroyed text_image with unused text_reference: #{image.id}, imageable_id: #{image.imageable.id}"
-        end
+        next if image.imageable.to_json.include?(image.text_reference)
+
+        image.destroy! if live_run
+        n_ti_destroyed += 1
+        puts " destroyed text_image with unused text_reference: #{image.id}, imageable_id: #{image.imageable.id}"
+      end
+
+      if n_ti_destroyed > 0 || n_li_destroyed > 0
+        record_by_tenant <<
+          "tenant_host: #{tenant.host}, " \
+          "tenant_id: #{tenant.id}, " \
+          "n_layout_images_destroyed: #{n_li_destroyed}, " \
+          "n_text_images_destroyed: #{n_ti_destroyed}"
```

Comment:
Details as string, not hash, as Sentry will print each key-value pair on newline, so this approach will avoid truncation by Sentry.